packages/config/eslint/index.js

```diff
@@ -8,7 +8,7 @@ const config = {
     "plugin:@typescript-eslint/recommended-requiring-type-checking",
   ],
   parser: "@typescript-eslint/parser",
-  plugins: ["@typescript-eslint", "import"],
+  plugins: ["@typescript-eslint", "import", "svelte3"],
```

Comment:
We may want to break down the eslint config as to not include rules/plugins that are not used in every package. 

https://eslint.org/docs/latest/use/configure/configuration-files#cascading-and-hierarchy

cc @juliusmarminge @t3dotgg

Reply:
Yes we can have multiple presets and then packages extends only what it needs:

https://github.com/t3-oss/create-t3-turbo/tree/main/packages/config/eslint

https://github.com/t3-oss/create-t3-turbo/blob/f73087c2422fd855c0e76ea69c66dcd6fed1e719/apps/nextjs/package.json#L46-L53

https://github.com/t3-oss/create-t3-turbo/blob/f73087c2422fd855c0e76ea69c66dcd6fed1e719/packages/api/package.json#L26-L31

I can do that tonight